ts/screens/main/MessagesScreen.tsx

```diff
@@ -1,24 +1,82 @@
-import { Container } from "native-base";
+import {
+  Body,
+  Container,
+  Content,
+  H1,
+  List,
+  Tab,
+  Tabs,
+  Text,
+  View
+} from "native-base";
 import * as React from "react";
-import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { connect } from "react-redux";
+import { CreatedMessageWithContent } from "../../../definitions/backend/CreatedMessageWithContent";
```

Comment:
CreatedMessageWithContent is Notification stuff. Messages use MessageWithContent.